actions/builder/update/entrypoint/'

```diff
@@ -0,0 +1,311 @@
+package main_test
```

Comment:
I'm a bit confused about the name of this file being `'` is there a reason for this or is it a typo?

Reply:
Yeah. Looks like this got checked in accidentally.